src/composables/maxAmount.ts

```diff
@@ -42,6 +42,7 @@ export interface MaxAmountOptions extends IDefaultComposableOptions {
  * considering the fee that needs to be paid.
  */
 export function useMaxAmount({ store, formModel }: MaxAmountOptions) {
+  // TODO - SW-791
```

Comment:
I don't like idea to introduce Jira's task number in the code. It is not obvious what suppose to be done in this file. The repo is public and all the todos can be fixed by any community member as well.